compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -104,11 +109,19 @@ static llvm::cl::opt<bool> clForceArmStreaming(
         "than SVE). Requires the +sme feature flag."),
     llvm::cl::init(false));
 
-static void addTileAndDistributePasses(OpPassManager &funcPassManager) {
-  funcPassManager.addPass(createTileAndDistributeToWorkgroupsPass());
-  funcPassManager.addPass(createCSEPass());
-  funcPassManager.addPass(createConvertToDestinationPassingStylePass());
-  funcPassManager.addPass(createFoldAffineMinInDistributedLoopsPass());
+// TODO: Enable `TileDispatchUsingForall` for every pipeline.
+static void
+addTileAndDistributePasses(OpPassManager &funcPassManager,
+                           bool enableTileDispatchUsingForall = false) {
```

Comment:
Please remove the default value. Because one of our goals is switching all the pipelines to use forall op. Explicitly writing it down help us track it better. We can remove the code when we find that all the uses are true.